src/tigerbeetle/inspect.zig

```diff
@@ -0,0 +1,1100 @@
+//! Decode a TigerBeetle data file without running a replica or modifying the data file.
+//! This tool adheres to the "be liberal in what you accept" side of Postel's Law.
```

Comment:
I'd maybe expand a bit on the why of the tool here. In parcicular, worth mentionning wherether the intended audience are the operators, or the developers of TigerBeetle.

In general, I feel like we could on average a couple of pararaphs of prose at the start of each file, as that's a very good way to "orient"  the reader in the codebase!